gsa/src/web/pages/results/details.js

```diff
@@ -102,14 +102,14 @@ DerivedDiff.propTypes = {
 const ResultDetails = ({className, links = true, entity}) => {
   const result = entity;
 
-  const {nvt} = result;
-  const {id: nvtId, tags, solution} = nvt;
+  const {information} = result;
+  const {id: infoId, tags, solution} = information;
 
-  const is_oval = hasValue(nvtId) && nvtId.startsWith('oval:');
-  const hasDetection = hasValue(result.detectionResult);
+  const is_oval = hasValue(infoId) && infoId.startsWith('oval:');
```

Comment:
all variables we are touching with code changes should be converted to camelCase. Could you change it to `isOval`?